core/src/components/searchbar/searchbar.ios.scss

```diff
@@ -155,6 +155,15 @@
   }
 }
 
+// Searchbar in Toolbar
+// -----------------------------------------
+
+:host-context(ion-toolbar) {
+  @include padding(1px, null, 15px, null);
```

Comment:
@brandyscarney Hello! Is this supposed to be `1px` for the top padding? Because updating to this version broke all our searchbars in toolbars. If I change it to `15px` it works fine as before.

Reply:
Yes, please see my comment here: https://github.com/ionic-team/ionic/issues/19502#issuecomment-537522707